availability/watcher.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2016: Mirantis Inc.
+# All Rights Reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import datetime as dt
+import json
+import logging
+import sys
+import threading
+import time
+
+import queue
+import requests
+
+from availability import config
+
+
+SERVICES = config.get_config().get("services")
```

Comment:
I am not fun of creating all this global variable. 

Why you can't create local variable in main, intialize them and send via arguments, code readability will be way bigger.